src/renderer/processes/multisigs/model/__tests__/multisigs-model.test.ts

```diff
@@ -18,37 +37,54 @@ describe('features/multisigs/model/multisigs-model', () => {
     ]);
   });
 
-  test('should not create a multisig we already have', async () => {
```

Comment:
Tests was not working at all 😢
Added proper handler for `interval` event via `fakeTimers`